script.js

```diff
@@ -0,0 +1,95 @@
+const hamberger = document.querySelector('.hamberger');
+const close = document.querySelector('.close');
+const body = document.querySelector('body');
+const menu = document.querySelector('.menu');
+const bodyOverlay = document.querySelector('.body-overlay');
+
+hamberger.addEventListener('click', () => {
+  close.classList.remove('hidden');
+  bodyOverlay.classList.remove('hidden');
+  menu.classList.add('hamberger-menu');
+  hamberger.classList.add('hidden');
+  body.classList.add('non-scroll');
+});
+
+close.addEventListener('click', () => {
+  close.classList.add('hidden');
+  bodyOverlay.classList.add('hidden');
+  menu.classList.remove('hamberger-menu');
+  hamberger.classList.remove('hidden');
+  body.classList.remove('non-scroll');
+});
+
+const speakers = [
+  {
+    name: 'Saeedeskandari',
+    title: 'CEO',
+    imgFeture: 'https://userphotos2.teacheron.com/1530294-07160.jpg',
+    description: 'I am fully expert in using AutoCAD, Revit and Excel, Because of years experience use of them in many projects.',
+  },
+  {
+    name: 'Kayla Snow',
+    title: 'AutoCAD and Revit Architect',
+    imgFeture: 'https://userphotos2.teacheron.com/1529707-03285.jpg',
+    description: 'career drafter teaching Revit and 2D AutoCAD skills',
+  },
+  {
+    name: 'Takeru Kondo',
+    title: 'BIM Specialist',
+    imgFeture: 'https://userphotos2.teacheron.com/1513636-15272.jpg',
+    description: 'I am licensed architect with over 10 years of experience in commercial architecture.',
+  },
+  {
+    name: 'Rizwan Razzak Ali',
+    title: 'CAD/CFD/FEA Expert',
+    imgFeture: 'https://userphotos2.teacheron.com/1511945-15254.jpeg',
+    description: 'I achieved many certificates from Dassault Systems, AutoDesk, and Bentley USA for excellent Design skills',
+  },
+  {
+    name: 'Emmanuel Kuchocha',
+    title: 'Architecture; Architectural Software; Revit',
+    imgFeture: 'https://userphotos2.teacheron.com/1510752-61473.jpg',
+    description: 'I am offering lessons in Autodesk Revit Architecture softwware as well as English lessons.',
+  },
+  {
```

Comment:
- [ ] Kindly make sure the mobile design for speakers stick as match as possible to the one of the template design. If you check the images below you'll noticed that two speakers are displayed by default, so I can suggest you implement a JavaScript feature that will allow you to show the rest of speakers.

| Template Design | Your Design |
| --------------------- | ----------------- |
| ![Screenshot from 2023-01-05 20-46-57](https://user-images.githubusercontent.com/83042742/210858070-f31a9471-7f14-4a99-a57e-cbc63448a2df.png) | ![Screenshot from 2023-01-05 20-51-42](https://user-images.githubusercontent.com/83042742/210858130-678e86a9-9744-4029-bc08-a93356bfb79f.png) |